actions/loadconfig.py

```diff
@@ -5,22 +5,31 @@ class NapalmLoadConfig(NapalmBaseAction):
     """Load configuration into network device via NAPALM
     """
 
-    def run(self, config_file, method, **std_kwargs):
+    def run(self, config_file, method, inline_transfer, **std_kwargs):
 
-        if not method:
-            method = 'merge'
-        else:
-            method = method.lower()
-            if method not in ["merge", "replace"]:
-                raise ValueError(('{} is not a valid load method, use: '
-                                  'merge or replace').format(method))
+        try:
+            if not method:
+                method = 'merge'
+            else:
+                method = method.lower()
+                if method not in ["merge", "replace"]:
+                    raise ValueError(('{} is not a valid load method, use: '
+                                      'merge or replace').format(method))
 
-        with self.get_driver(**std_kwargs) as device:
+            with self.get_driver(**std_kwargs) as device:
+                # inline_transfer: If set it becomes True, else False
+                device.inline_transfer = inline_transfer
 
-            if method == "replace":
-                device.load_replace_candidate(filename=config_file)
-            else:
-                device.load_merge_candidate(filename=config_file)
+                if method == "replace":
+                    device.load_replace_candidate(filename=config_file)
+                else:
+                    device.load_merge_candidate(filename=config_file)
+
+                device.commit_config()
+
+        except Exception, e:
```

Comment:
This block was removed in #34. It looks like you have the commits from that incorporated already, so maybe something else happened in your editor? Anyways, please ensure this is removed per the work in #34

Reply:
Yep - will take care of this. 

I'm not quite sure what happened with this. I pulled the latest version to work on and it had the try/except statements in. When looking on the Web UI, they're not there. Will try again and see what's going on.